src/packagedcode/gemfile_lock.py

```diff
@@ -153,6 +155,12 @@ def __init__(self, name=None, version=None, platform=None):
         # a map of direct dependent Gems, keyed by name
         self.dependencies = OrderedDict()
 
+    def __lt__(self,other): 
```

Comment:
We can't able to sort Gem object So this method needed to compare . Otherwise it gives `TypeError: '<' not supported between instances of 'Gem' and 'Gem' `

Reply:
:+1: